examples/ice-restart/main.go

```diff
@@ -15,7 +15,25 @@ func doSignaling(w http.ResponseWriter, r *http.Request) {
 	var err error
 
 	if peerConnection == nil {
-		if peerConnection, err = webrtc.NewPeerConnection(webrtc.Configuration{}); err != nil {
+		m := webrtc.MediaEngine{}
```

Comment:
Maybe better to do this in another example (and that can come in another PR!)

I just don't want people to have to deal with multiple concepts at once :/ but I think duplication is fine!

Reply:
Makes total sense. I added another example to `examples/ice-tcp` and reverted this one. Now users don't need to write rules for blocking UDP packets!